crypto/bio/bss_file.c

```diff
@@ -427,10 +428,14 @@ static long MS_CALLBACK file_ctrl(BIO *b, int cmd, long num, void *ptr)
         b->shutdown = (int)num;
         break;
     case BIO_CTRL_FLUSH:
-        if (b->flags & BIO_FLAGS_UPLINK)
-            UP_fflush(b->ptr);
-        else
-            fflush((FILE *)b->ptr);
+        st = b->flags & BIO_FLAGS_UPLINK
+                ? UP_fflush(b->ptr) : fflush((FILE *)b->ptr);
```

Comment:
I don't quite understand why `if` had to be rewritten as conditional expression, as it appears less readable now. It would be equally readable if was formatted as
```
        st = b->flags & BIO_FLAGS_UPLINK ? UP_fflush(b->ptr)
                                         : fflush((FILE *)b->ptr);
```
but all right since it's effectively as cherry-pick...